code/modules/exosuits/premade/light.dm

```diff
@@ -0,0 +1,90 @@
+/mob/living/exosuit/premade/light
```

Comment:
This and the sleek exosuit seem to be almost the same thing, just with slight different descriptions/names, are they differernt icons?

Reply:
Different icons, yes.